promql/comparer/comparer.go

```diff
@@ -169,3 +170,23 @@ func addDropResultLabelsOptions(queryTweaks []*config.QueryTweak, options *cmp.O
 		}
 	}
 }
+
+func addCaseInsensitiveCompareOptions(queryTweaks []*config.QueryTweak, options *cmp.Options) {
+	for _, rt := range queryTweaks {
+		if rt.IgnoreCase {
+			*options = append(
+				*options,
+				// Translate sample values into lowercase.
```

Comment:
"Sample value" usually refers only to the numeric float64 part of a sample, so I suggest changing it to:

```suggestion
				// Translate metric names and labels into lowercase.
```